flyteadmin/pkg/errors/errors.go

```diff
@@ -112,31 +112,30 @@ func NewIncompatibleClusterError(ctx context.Context, errorMsg, curCluster strin
 }
 
 func compareJsons(jsonArray1 jsondiff.Patch, jsonArray2 jsondiff.Patch) []string {
-	results := []string{}
+	var results []string
```

Comment:
Should add a nil check if initializing a zero value(which is nil) slice is intended?